### ISSUE TITLE
Not working on latest Laravel 5.6 version

### ISSUE THREAD

**Issue by ronjunevaldoz** [michaeldrennen/Geonames#8]
```
Problem 1
    - The requested package phpunit/phpunit (locked at 6.5.6, required as ~7.0) is satisfiable by phpunit/phpunit[6.5.6] but these conflict with your requirements or minimum-stability.
  Problem 2
    - Installation request for michaeldrennen/geonames ^1.0 -> satisfiable by michaeldrennen/geonames[v1.0.0].
    - michaeldrennen/geonames v1.0.0 requires ext-intl * -> the requested PHP extension intl is missing from your system.
  Problem 3
    - Installation request for sebastian/diff (locked at 2.0.1) -> satisfiable by sebastian/diff[2.0.1].
    - phpunit/phpunit 7.0.0 requires sebastian/diff ^3.0 -> satisfiable by sebastian/diff[3.0.0].
    - phpunit/phpunit 7.0.1 requires sebastian/diff ^3.0 -> satisfiable by sebastian/diff[3.0.0].
    - phpunit/phpunit 7.0.2 requires sebastian/diff ^3.0 -> satisfiable by sebastian/diff[3.0.0].
    - Conclusion: don't install sebastian/diff 3.0.0
    - Installation request for phpunit/phpunit ~7.0 -> satisfiable by phpunit/phpunit[7.0.0, 7.0.1, 7.0.2].
```

**Comment by michaeldrennen** [comment]
Those errors don't have anything to do with Laravel 5.6.

(I just created a fresh Laravel project at v5.6.18)

Some other package you have in that project is conflicting with the versions required by my Geonames library.

**Comment by carlituxman** [comment]
I've just tried in a fresh version with laravel 5.6.26 and has error with ext-intl :

Problem 1
    - michaeldrennen/geonames v1.0.8 requires ext-intl * -> the requested PHP extension intl is missing from your system.
    - michaeldrennen/geonames v1.0.7 requires ext-intl * -> the requested PHP extension intl is missing from your system.
.......

**Comment by carlituxman** [comment]
solved updating php version 7.1.13 to 7.1.19
